### PR TITLE
Auto watch fix

### PR DESCRIPTION
**Pull request by Justin-Holmes**

### CODE REVIEW

File: src/selectors/profile.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const selectRegistrationId = state => get(state, 'profile.registrationId'
 export const selectShortBio = state => get(state, 'profile.shortBio', '')
 export const selectUsername = state => get(state, 'profile.username')
 export const selectWebOnboardingVersion = state => get(state, 'profile.webOnboardingVersion')
+export const selectAllowsAutoWatch = state => get(state, 'profile.hasAutoWatchEnabled')
```
Comment (mkitt) [inline src/selectors/profile.js]: We should call this one `selectHasAutoWatchEnabled`. The pattern here is just prefixing the word `select` to the property name as it appears in the store.

File: src/actions/comments.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import * as MAPPING_TYPES from '../constants/mapping_types'
 import * as api from '../networking/api'
 import { resetEditor } from '../actions/editor'
 
-export function createComment(body, editorId, postId) {
+export function createComment(body, editorId, postId, hasAutoWatchEnabled) {
```
Comment (mkitt) [inline src/actions/comments.js]: Can we sort these alphabetically 🔤 ?

File: src/components/editor/Editor.js [hunk 3]
```diff
@@ -1,7 +1,10 @@
 import React, { Component, PropTypes } from 'react'
 import { connect } from 'react-redux'
 import { selectIsLoggedIn } from '../../selectors/authentication'
-import { selectIsOwnPage } from '../../selectors/profile'
+import {
+  selectIsOwnPage,
+  selectAllowsAutoWatch,
+} from '../../selectors/profile'
```
Comment (mkitt) [inline src/components/editor/Editor.js]: Can we sort these alphabetically 🔤 ?

File: src/components/editor/Editor.js [hunk 4]
```diff
@@ -46,6 +49,7 @@ function mapStateToProps(state, props) {
     post: selectPostFromPropsPostId(state, props),
     isOwnPage: selectIsOwnPage(state),
     isOwnPost: selectIsOwnPost(state, props),
+    allowsAutoWatch: selectAllowsAutoWatch(state),
   }
```
Comment (mkitt) [inline src/components/editor/Editor.js]: Can we sort these alphabetically 🔤 ?

File: src/components/editor/Editor.js [hunk 5]
```diff
@@ -64,6 +68,7 @@ class Editor extends Component {
     post: PropTypes.object,
     shouldLoadFromState: PropTypes.bool,
     shouldPersist: PropTypes.bool,
+    allowsAutoWatch: PropTypes.bool,
```
Comment (mkitt) [inline src/components/editor/Editor.js]: Can we sort these alphabetically 🔤 ?

File: src/components/editor/Editor.js [hunk 6]
```diff
@@ -90,13 +95,13 @@ class Editor extends Component {
   }
 
   submit = (data) => {
-    const { comment, dispatch, isComment, isOwnPage, onSubmit, post } = this.props
+    const { comment, dispatch, isComment, isOwnPage, onSubmit, post, allowsAutoWatch } = this.props
     if (isComment) {
```
Comment (mkitt) [inline src/components/editor/Editor.js]: Can we sort these alphabetically 🔤 ?

File: src/reducers/experience_updates/comments.js [hunk 7]
```diff
@@ -21,13 +21,13 @@ methods.updateCommentsCount = (newState, postId, delta) => {
 }
 
 methods.addOrUpdateComment = (newState, action) => {
-  const { model, postId, response } = action.payload
+  const { model, postId, response, hasAutoWatchEnabled } = action.payload
   const post = newState[MAPPING_TYPES.POSTS][postId]
```
Comment (mkitt) [inline src/reducers/experience_updates/comments.js]: Can we sort these alphabetically 🔤 ?

**Comment by rynbyjn** [comment]
Not totally sure what is up with node stuff..

**Pull request merged**